### DISCUSSION TITLE
Is the temperature sensor of the echo still populated?

### DISCUSSION THREAD

**Discussion by ilu2k21** [alandtse/alexa_media_player#2534] [General]
Hi there,
reinstalled alexa media player today, after not working anymore. Everything went fine, but im missing the temperature sensor entity.

**Comment by danielbrunt57** [comment]
You need to enable:
![image](https://github.com/user-attachments/assets/e7ff7c5d-815d-4f7c-8dde-b29b6ff58b7f)

**Reply by ilu2k21** [reply]
ah that helped thanks

**Comment by danielbrunt57** [comment]
Issue solved
